steps/addTextWidgetTo.js

```diff
@@ -25,12 +25,16 @@ module.exports = ({selector, text} = {}) => {
       client.pause(200);
       client.clickWhenReady(richTextBtnSelector);
       client.pause(2000);
-      client.execute(function (content) {
-          const ckeditorInstance = _.find(CKEDITOR.instances);
+      client.execute(function (content, selector) {
+          // Pick out the CKEditor instance we actually
+          // care about, avoiding updating all of them
+          const id = $(selector).attr('id');
```

Comment:
For bc you should skip the find-the-id-and-use-that step if the second parameter is not passed. Otherwise existing tests will probably fail.

Reply:
the `selector` in this case is always passed, as it's a required parameter to the step itself.